src/app/modules/main/view.nim

```diff
@@ -111,12 +114,18 @@ QtObject:
     self.tmpCommunityId = ""
     if(communityVariant.isNil):
       return newQVariant()
-    
+
     return communityVariant
 
   proc getChatSectionModule*(self: View): QVariant {.slot.} =
     return self.delegate.getChatSectionModule()
 
+  proc getAllChatsModel*(self: View): QVariant {.slot.} =
+    return self.delegate.getAllChatsModel()
```

Comment:
This would be better as a QtProperty. Example:
```nim
  QtProperty[QVariant] allChatsModel:
    read = getAllChatsModel
    notify = allChatsModelChanged # this signal would need to be added
```